ci-operator/step-registry/ipi/conf/inframachineset/ipi-conf-inframachineset-commands.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if test ! -f "${KUBECONFIG}"
+then
+	echo "No kubeconfig, can not continue."
+	exit 0
+fi
+
+# Download jq
+mkdir /tmp/bin
+export PATH=$PATH:/tmp/bin
+curl -sL https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 > /tmp/bin/jq
+chmod ug+x /tmp/bin/jq
+
+# Create infra machineconfigpool
+oc create -f - <<EOF
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: infra
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,infra]}
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/infra: ""
+EOF
+
+# Get machineset name to generate a generic template
+ref_machineset_name=$(oc get machinesets -n openshift-machine-api |grep worker |awk 'NR==1{print $1}')
```

Comment:
Shifting the `awk` magic into `oc`:

```console
$ oc -n openshift-machine-api get -o 'jsonpath={range .items[*]}{.metadata.name}{"\n"}{end}' machinesets | grep worker | head -n1
ci-ln-f3b1f8t-f76d1-x8v7h-worker-a
```

Not sure if that's all that much of a simplification or not...